app.go

```diff
@@ -25,20 +23,24 @@ import (
 	"sync"
 	"time"
 
-	utils "github.com/gofiber/utils"
-	colorable "github.com/mattn/go-colorable"
-	isatty "github.com/mattn/go-isatty"
-	fasthttp "github.com/valyala/fasthttp"
+	"github.com/gofiber/fiber/v2/utils"
+	"github.com/gofiber/fiber/v2/utils/colorable"
+	"github.com/gofiber/fiber/v2/utils/isatty"
+	"github.com/valyala/fasthttp"
 )
 
 // Version of current package
-const Version = "1.14.6"
+const Version = "2.0.0"
```

Comment:
Interesting article, but I think it would overcomplicate things. We don't mind bumping the version on each release, maybe this is something we could implement in the future after we released v2.